core/src/main/java/discord4j/core/object/entity/channel/ThreadChannel.java

```diff
@@ -36,17 +37,41 @@
 import java.time.Instant;
 import java.time.format.DateTimeFormatter;
 import java.util.EnumSet;
+import java.util.List;
 import java.util.Objects;
 import java.util.Optional;
 import java.util.function.Function;
-
+import java.util.stream.Collectors;
```

Comment:
Remove the break line

Reply:
WDYM ? 
The only line break is between external libs and Java standard lib.
I think that's a diff issue.

![image](https://github.com/user-attachments/assets/aae03c1e-7395-438c-a5ae-3935f551f162)